### PR TITLE
MINOR: move TimeRatio to server-common

### PR DESCRIPTION
**Pull request by jeffkbkim**
Move TimeRatio to `org.apache.kafka.server.metrics` package under `server-common`

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by dajac** [comment]
I am not sure about this one. `org.apache.kafka.common.metrics.stats` is part of public API so moving `TimeRatio` there would effectively make it part of our public API. We would need a KIP for this. Have you considered moving it to `server-common`?

File: raft/src/test/java/org/apache/kafka/raft/internals/TimeRatioTest.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 package org.apache.kafka.raft.internals;
 
 import org.apache.kafka.common.metrics.MetricConfig;
+import org.apache.kafka.server.metrics.TimeRatio;
```
Comment (ijuma) [inline raft/src/test/java/org/apache/kafka/raft/internals/TimeRatioTest.java]: This test should move too, right?

Comment (jeffkbkim) [reply raft/src/test/java/org/apache/kafka/raft/internals/TimeRatioTest.java]: thanks for the catch! moved the test

**Comment by jeffkbkim** [comment]
closing as it will not be used in a package outside of `raft`

**Pull request closed without merging**